plugins/com.aptana.samples.ui/META-INF/MANIFEST.MF

```diff
@@ -14,10 +14,12 @@ Require-Bundle: com.aptana.jetty.util.epl,
  com.aptana.git.ui,
  com.aptana.projects,
  com.aptana.configurations,
- com.aptana.portal.ui
+ com.aptana.portal.ui,
+ com.aptana.scripting;bundle-version="3.1.0"
```

Comment:
We shouldn't reference the bundles using the version. That might break the dependency chain in the future once we change the bundle versions.